source/developers/transform-data-with-object-lambda.rst

```diff
@@ -0,0 +1,242 @@
+.. _developers-object-lambda:
+
+=================================
+Transform Data with Object Lambda
+=================================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+The following documentation covers transforming the contents of an object with MinIO's Object Lambda.
+Uses for Object Lambda include redacting personally identifable information (PII) and enriching data with information from other sources.
+
+This documentation assumes the following:
+
+- An existing :ref:`MinIO <minio-installation>` deployment
+- Working Python (3.8+) and Golang (??) development environments
+- :doc:`The MinIO Go SDK </developers/go/minio-go>` installed
+  
+-------
+Example
+-------
+
+This example creates a Lambda handler in Python, invokes it from a Go function, and retrieves the transformed object with ``curl``.
+
+
+Create a Lambda Handler
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The sample handler, written in Python, retrieves the target object using a `presigned URL <https://min.io/docs/minio/linux/developers/go/API.html#presigned-operations>`__. It then transforms the object's contents and returns the new text to the caller.
```

Comment:
"returns the new text to the caller" is vague. What is a better way to describe the path back to who/what requested the transformed object?